frontend/src/tests/api.test.js

```diff
@@ -83,6 +83,9 @@ describe("Mock API tests", () => {
     describe("`/admin/assignments` tests", () => {
         assignmentsTests(mockAPI);
     });
+    describe("offers tests", () => {
```

Comment:
`"offers tests"` should most likely match the `"/foo/bar"` syntax of other tests

Reply:
good point, I ll change that

Reply:
so all the routes that related to offers are  under assignments for example: `/admin/assignments/${assignment_id}/active_offer`
I don't know if I should just leave it as `offer tests` or change to `/xxx/yyy` since it's technically part of assignments. What do you think, @ilikeballoons ? and what do you suggest, @siefkenj ?

Reply:
Put the full route in there. The API is interconnected, and that's okay. It's just how it is.

Reply:
sure, np